content/docs/rest-helix/users-get.md

```diff
@@ -57,7 +57,10 @@ resultList.getUsers().forEach(user -> {
 {{</code>}}
 {{<code Kotlin>}}
 ```kotlin
-
+val resultList = twitchClient!!.helix.getUsers(null, listOf("149223493"), null).execute()
+resultList.users.forEach(Consumer { user: User? ->
+	println(user)
+})
```

Comment:
```suggestion
resultList.users.forEach { user ->
	println(user)
}
```

Reply:
Consumer have a [SAM interfaces](https://kotlinlang.org/docs/fun-interfaces.html) in code.